administrator/language/en-GB/en-GB.com_joomlaupdate.ini

```
@@ -43,17 +43,32 @@ COM_JOOMLAUPDATE_VIEW_COMPLETE_HEADING="Joomla Version Update Status"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_COMPATIBLE_UPGRADE_WARNING="Extensions marked with <span class='label label-warning'>X.X.X</span> have an extension upgrade available for the current version of Joomla which is not marked as compatible with the upgraded version of Joomla.  You should contact the extension developer for more information."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DESCRIPTION_BREAK="Extensions marked with <span class='label label-important'>No</span> or <span class='label'>Missing Compatibility Tag</span> might break your website. Please consult with the developer before upgrading."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DESCRIPTION_MISSING_TAG="Extensions marked with <span class='label'>Missing Compatibility Tag</span> indicate the developer has not included <a href='https://docs.joomla.org/Special:MyLanguage/Deploying_an_Update_Server' target='_blank' rel='noopener noreferrer'>compatibility information.</a>"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DESCRIPTION_UPDATE_REQUIRED="Extensions marked with <span class='label label-warning'>Yes (X.X.X)</span> might require an update."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DIRECTIVE="Directive"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DOWNLOAD_IN_PROGRESS="Downloading update file. Please wait ..."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_FTP_DIRECTORY="FTP Root"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_RUNNING_PRE_UPDATE_CHECKS="Running Pre-Update Checks"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_RUNNING_PRE_UPDATE_CHECKS_NOTES="Please be patient whilst we run the pre-update checks on these extensions."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE="Extensions Probably Compatible"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>The extension developer has provided us with information to suggest that the currently installed version is compatible with the available upgraded version of Joomla.</p><p id='upgradeyellowwarning' class='hidden'>Please note that if you see a version highlighted as <span class='label label-warning'>n.n.n</span> then the extension developer is offering a newer version of the extension for your current version of Joomla than they do for the new version of Joomla.  You should check with the extension developer if this is correct before you ugprade Jomla.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPGRADES_TO_BE_COMPATIBLE="Extensions Requiring Upgrades to be Compatible"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPGRADES_TO_BE_COMPATIBLE_NOTES="<p>The extension developer is offering a new version of the extension that they say is compatible with the available upgraded version of Joomla.</p><p id='upgradeorangewarning' class='hidden'>Please take extra care if this upgraded version of the extension is not also listed as compatible with your current version of Joomla.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION="Update Server Offers No Compatible Version"
```

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION="Update Server offers no compatible version"
```

I'm not 100% sure on this one maybe @brianteeman can give advise on the capalization of the headings?

---

The other headings are long word capitalised  - so if we change this we should change the others.